plugins/elantp/fu-elantp-hid-device.c

```diff
@@ -63,9 +65,10 @@ fu_elantp_hid_device_probe(FuDevice *device, GError **error)
 		return FALSE;
 	}
 
+	did = fu_udev_device_get_model(FU_UDEV_DEVICE(device));
```

Comment:
you can move this up, e.g. `guint16 device_id = fu_udev_device_get_model(FU_UDEV_DEVICE(device));;`

Reply:
Done